ERROR: 2022/12/03 20:54:04 D:/Documentos/Proyecto SI/system_school_back/api/handle/report/handler.go:22: couldn't bind model login: Unprocessable Entity
ERROR: 2022/12/03 23:23:18 D:/Documentos/Proyecto SI/system_school_back/pkg/entity/file/application_service.go:30: 8aade556-f21f-4c30-905c-40d214f2b301  - couldn't create File : open : The system cannot find the file specified.
ERROR: 2022/12/03 23:25:00 D:/Documentos/Proyecto SI/system_school_back/pkg/entity/file/application_service.go:30: 8aade556-f21f-4c30-905c-40d214f2b301  - couldn't create File : open : The system cannot find the file specified.
ERROR: 2022/12/03 23:29:42 D:/Documentos/Proyecto SI/system_school_back/pkg/entity/file/application_service.go:30: aa5acb59-6568-4e0e-a65b-934a5b7f01dc  - couldn't create File : open : The system cannot find the file specified.
ERROR: 2022/12/03 23:34:08 D:/Documentos/Proyecto SI/system_school_back/pkg/entity/file/application_service.go:30: db8293da-43cf-4533-a88b-f95db96dbbfd  - couldn't create File : open : The system cannot find the file specified.
ERROR: 2022/12/03 23:48:43 D:/Documentos/Proyecto SI/system_school_back/pkg/entity/file/application_service.go:30: 258835bc-0706-4c50-aa71-5a8145f744af  - couldn't create File : open : The system cannot find the file specified.
ERROR: 2022/12/03 23:49:05 D:/Documentos/Proyecto SI/system_school_back/api/handle/users/handler.go:241: couldn't bind model login: json: expected '"' at the beginning of a string value: full_path:"files/0001_13.png"}
ERROR: 2022/12/03 23:49:22 D:/Documentos/Proyecto SI/system_school_back/pkg/entity/file/application_service.go:30: 258835bc-0706-4c50-aa71-5a8145f744af  - couldn't create File : open : The system cannot find the file specified.
ERROR: 2022/12/05 17:31:33 D:/Documentos/Proyecto SI/system_school_back/pkg/auth/users/application_service.go:90: 9eeb322e-d78c-4cf4-b669-befc49b10454  - error in saved iamge: El usuario ya existe!
ERROR: 2022/12/05 17:31:33 D:/Documentos/Proyecto SI/system_school_back/api/handle/doctypes/handler.go:32: couldn't create user: El usuario ya existe!
ERROR: 2022/12/05 17:31:36 D:/Documentos/Proyecto SI/system_school_back/pkg/auth/users/application_service.go:90: 9eeb322e-d78c-4cf4-b669-befc49b10454  - error in saved iamge: El usuario ya existe!
ERROR: 2022/12/05 17:31:36 D:/Documentos/Proyecto SI/system_school_back/api/handle/doctypes/handler.go:32: couldn't create user: El usuario ya existe!
